guide/asynchronous-invocation/src/stock_quote_summary_service/Module.md

```diff
@@ -0,0 +1,3 @@
+# Hello World
+
+Prints "hello world" to command line output
```

Comment:
```suggestion
Prints "hello world" as the output in the command line.

Reply:
Auto generated file from `ballerina add` command.

Reply:
Will be fixed via https://github.com/ballerina-platform/ballerina-lang/pull/19273.